[CF1] Overview revamp (#21611)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/index.mdx

    content: Overview
---

import {
	CardGrid,
	Description,
	Feature,
  GlossaryTooltip,
	LinkButton,
	LinkTitleCard,
	Plan,
	RelatedProduct,
	Render,
} from "~/components";

<Description>

> Secure your organization with Cloudflare Zero Trust - a cloud security model that replaces legacy perimeters with Cloudflare’s global network.

Reviewer: ```suggestion
Secure your organization with Cloudflare Zero Trust -- a cloud security model that replaces legacy perimeters with Cloudflare’s global network.
```

</Description>

<Plan type="all" />

> **Cloudflare Zero Trust** is part of **Cloudflare One**, our name for the Secure Access Service Edge (SASE) platform that protects enterprise applications, users, devices, and networks.

Reviewer 2: ```suggestion
Cloudflare Zero Trust is part of Cloudflare One our name for the Secure Access Service Edge (SASE) platform that protects enterprise applications, users, devices, and networks.
```

Reviewer 2: (According to our style guide, we use bold for UI elements.)

> By progressively adopting Cloudflare One, organizations can move away from a patchwork of hardware appliances and point solutions, and instead consolidate security and networking through a unified control plane that includes products like **Cloudflare Zero Trust Network Access (ZTNA)**, **Secure Web Gateway (SWG)**, **Cloudflare Tunnel**, **Data Loss Prevention (DLP)**, **Remote Browser Isolation (RBI)**, **Cloud Access Security Broker (CASB)**, and **Email Security**.

Reviewer 2: Did we officially change the name of Cloudflare Access to Cloudflare Zero Trust Network Access?

Reviewer 3: No, but I think the marketing site uses them interchangeably sometimes. Since everything else in this list is the product name vs. use case, I lean toward sticking with 'Access.'

Refer to our [SASE reference architecture](/reference-architecture/architectures/sase/) to learn how to plan, deploy, and manage SASE architecture with Cloudflare.

<LinkButton variant="primary" href="/cloudflare-one/setup/">
	Get started
</LinkButton> <LinkButton
	variant="secondary"
	href="https://one.dash.cloudflare.com/"
>
	Zero Trust dashboard
</LinkButton><LinkButton
	variant="secondary"
	href="/cloudflare-one/implementation-guides/"
>
	Implementation guides

> </LinkButton>

Reviewer: Is this button visible enough of a callout for the implementation guides? This is especially important for users new to ZT.

## Products

<RelatedProduct header="Access (Zero Trust Network Access - ZTNA)" href="/cloudflare-one/policies/access/" product="access">

Authenticate users accessing your applications, seamlessly onboard third-party users, and log every event and request.

</RelatedProduct>

<RelatedProduct header="Cloudflare Tunnel" href="/cloudflare-one/connections/connect-networks/" product="zero-trust-networks">

> Securely connect your resources to Cloudflare without exposing a public IP by using Cloudflare Tunnel, which establishes outbound-only connections from your infrastructure to Cloudflare’s global network via the lightweight `cloudflared` daemon.

Reviewer 2: We may need to change this description once [WARP Connector](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/private-net/warp-connector/) goes GA.

</RelatedProduct>

<RelatedProduct header="Secure Web Gateway (SWG)" href="/cloudflare-one/policies/gateway/" product="gateway">

> Inspect DNS, network, HTTP, and egress traffic to enforce your company's Acceptable Use Policy (UAP), block risky sites with custom blocklists and threat intelligence, and enhance visibility and protection across SaaS applications.

Reviewer: ```suggestion
Inspect and filter DNS, network, HTTP, and egress traffic to enforce your company's Acceptable Use Policy (UAP), block risky sites with custom blocklists and threat intelligence, and enhance visibility and protection across SaaS applications.
```

</RelatedProduct>

<RelatedProduct header="WARP" href="/cloudflare-one/connections/connect-devices/warp/" product="warp-client">

> Protect corporate devices by securely and privately routing traffic through Cloudflare’s global network while enforcing advanced web filtering and threat protection.

Reviewer 2: Technically the web filtering and threat protection is done by Gateway and DLP and not WARP. I would instead emphasize that WARP can enforce device posture rules.

</RelatedProduct>

<RelatedProduct header="Browser Isolation (RBI)" href="/cloudflare-one/policies/browser-isolation/" product="browser-isolation">

> Mitigate the impact of attacks by executing all browser code in the cloud, and deliver a seamless, lightning-fast end user experience.

Reviewer 2: nit: the last part of this sentence sounds like marketing speak

</RelatedProduct>

<RelatedProduct header="Cloud Access Security Broker (CASB)" href="/cloudflare-one/applications/casb/" product="casb">

> Protect users and sensitive data at rest in SaaS applications, detect insider threats and unsanctioned application usage, also known as <GlossaryTooltip term="shadow IT" link="https://www.cloudflare.com/learning/access-management/what-is-shadow-it/">shadow IT</GlossaryTooltip>, and ensure best practices to prevent data leaks and compliance violations.

Reviewer: ```suggestion
Protect users and sensitive data at rest in SaaS applications and cloud environments, detect insider threats and unsanctioned application usage, also known as <GlossaryTooltip term="shadow IT" link="https://www.cloudflare.com/learning/access-management/what-is-shadow-it/">shadow IT</GlossaryTooltip>, and ensure best practices to prevent data leaks and compliance violations.
```

Reviewer: We should also call out that CASB detects misconfigurations within SaaS apps and cloud environments.

Reviewer 3: I'd also remove mention of shadow IT here, our shadow IT as it exists in the dash today is more closely related to Gateway.

</RelatedProduct>

<RelatedProduct header="Data Loss Prevention (DLP)" href="/cloudflare-one/policies/data-loss-prevention/" product="data-loss-prevention">

> Detect sensitive data as it moves to and from SaaS applications, use predefined DLP profiles to get started quickly, and log or block DLP matches.

Reviewer 3: Maybe reference both for clarity: "Detect sensitive data within your web traffic and SaaS applications."

</RelatedProduct>

<RelatedProduct header="Email Security" href="/cloudflare-one/email-security/" product="email-security">

Configure policies to manage your inbox, automatically move emails based on disposition, and use screen criteria to investigate messages.

</RelatedProduct>

<RelatedProduct header="Digital Experience Monitoring (DEX)" href="/cloudflare-one/insights/dex/" product="dex">

Monitor device, network, and application performance across your Zero Trust organization.

</RelatedProduct>

---

## More resources

<CardGrid>

<LinkTitleCard

> 	title="Learning Paths"

Reviewer 2: ```suggestion
	title="Videos"
```

	href="/learning-paths/sase-overview-course/series/evolution-corporate-networks-1/"
	icon="pen"
>

> 	New to Zero Trust and SASE? Get started with the SASE introductory learning path.

Reviewer 2: ```suggestion
	New to Zero Trust and SASE? Get started with the SASE introductory video series.
```

</LinkTitleCard>

<LinkTitleCard
	title="Reference architecture"
	href="/reference-architecture/architectures/sase/"
	icon="open-book"
>
	Explore our reference architecture to learn how to evolve your network and security architecture to our SASE platform.
</LinkTitleCard>

<LinkTitleCard
	title="Plans"
	href="https://www.cloudflare.com/plans/zero-trust-services/"
	icon="seti:shell"
>
	Cloudflare Zero Trust offers both Free and Paid plans. Access to certain features depends on a customer's plan type.
</LinkTitleCard>

<LinkTitleCard title="Limits" href="/cloudflare-one/account-limits/" icon="document">

	Learn about account limits. These limits may be increased on Enterprise accounts.

</LinkTitleCard>

<LinkTitleCard
	title="Developer Discord"
	href="https://discord.cloudflare.com"
	icon="discord"
>

> 	Connect with Cloudflare developers on Discord to ask questions, and share what you

Reviewer 2: How active is the Discord channel for Zero Trust? Should we point users to [Cloudflare Community](https://community.cloudflare.com/) instead?
